api/src/controllers/events.ts

```diff
@@ -43,17 +60,42 @@ const eventsController = (clients: Clients, config: Config): EventsHandler => {
     const { query: queryString, ...params } = req.query;
     const sort = sortValidator(params)?.[0];
     const sortOrder = sortOrderValidator(params)?.[0];
-
+    const aggregations = aggregationsValidator(params);
     const sortKey =
       sort === "times.startDateTime" ? "query.times.startDateTime" : "_score";
 
+    const initialAggregations = ifDefined(aggregations, (requestedAggs) =>
+      pick(eventsAggregations, requestedAggs)
+    );
+
+    const postFilters = pickFiltersFromQuery(
+      ["format", "audience", "interpretation", "isOnline"],
```

Comment:
possible overkill and I know we don't do it in articles either, but I wonder if we couldn't make this array a constant as it's also used on line 52 and it make might it easier to not miss when adding a new one?

Reply:
They happen to be the same strings but they refer to different things: filters and aggregations. 
They're the same because all our filters are paired with aggregations but it might not always be the case so I'd rather keep them separated

Reply:
Thank you, that makes sense!